README.md

```diff
@@ -12,27 +12,27 @@
 
 ### Redundancy
 
-Akubra is a simple solution to keep an independent S3 storages in sync - almost
+Akubra is a simple solution to keep independent S3 storages in sync - almost
 realtime, eventually consistent.
 
-Keeping synchronized storage clusters, which handles great volume of new objects
-(about 300k obj/h), is the most efficient by feeding them with all incoming data
-at once. That's what Akubra does, with a minimum memory and cpu footprint.
+Keeping synchronized storage clusters, which handle great volume of new objects
+(about 300k obj/h), is most efficient by feeding them with all incoming data
+at once. That's what Akubra does, with a minimal memory and cpu footprint.
 
-Synchronizing S3 storages offline is almost impossible with a high volume traffic.
+Synchronizing S3 storages offline is almost impossible with a high volume of traffic.
 It would require keeping track of new objects (or periodical bucket listing),
-downloading and uploading them to other storage. It's slow, expensive and hard
+downloading and uploading them to the other storage. It's slow, expensive and hard
```

Comment:
Not sure if "the" is necessary here.

Reply:
I think it's ok here, because we point at a particular storage, if we would put here `other storages` we would omit the `the`.